src/mds/OpenFileTable.cc

```diff
@@ -31,6 +31,24 @@ static ostream& _prefix(std::ostream *_dout, MDSRank *mds) {
   return *_dout << "mds." << mds->get_nodeid() << ".openfiles ";
 }
 
+OpenFileTable::OpenFileTable(MDSRank *m) : mds(m) {
+  stringstream ss_name;
+  ss_name << "mds." << m->get_nodeid() << "-open_files_table";
+
+  PerfCountersBuilder b(mds->cct, ss_name.str(), l_oft_first, l_oft_last);
```

Comment:
ss_name.str() should be a static string literal. We don't want to have it depend on the rank because then `jq` JSON queries would be impossible.

I suggest just using `oft` for `OpenFileTable`.

Reply:
done